### PR TITLE
[AzureMonitor] remove features before stable release

### PR DESCRIPTION
**Pull request by TimothyMothra**
Removing certain logging features before stable release.
These features will be added back for an immediate beta release.

## Changes
- change LogExporter to `internal`
- remove LogFiltering from Distro
- remove AzureLogForwarding from Distro

### CODE REVIEW

File: sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/OpenTelemetryBuilderExtensions.cs [hunk 1]
```diff
@@ -169,16 +169,17 @@ public static OpenTelemetryBuilder UseAzureMonitor(this OpenTelemetryBuilder bui
...
+                        // TODO: WILL RE-ENABLE IN NEXT BETA
+                        //if (enableLogSampling)
+                        //{
+                        //    var azureMonitorExporterOptions = new AzureMonitorExporterOptions();
+                        //    azureMonitorOptions.SetValueToExporterOptions(azureMonitorExporterOptions);
+                        //    loggingOptions.AddProcessor(new LogFilteringProcessor(new AzureMonitorLogExporter(azureMonitorExporterOptions)));
+                        //}
+                        //else
+                        //{
+                        //    loggingOptions.AddAzureMonitorLogExporter(o => azureMonitorOptions.SetValueToExporterOptions(o));
```
Comment (rajkumar-rangaraj) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/OpenTelemetryBuilderExtensions.cs:181]: how does distro export logs with these commented out?

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,8 @@
 ### Other Changes
 
 * Includes all changes from 1.3.0-beta.1 and 1.3.0-beta.2
+* Changed `AzureMonitorLogExporter` to be internal.
```
Comment (vishweshbankwar) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:8]: clarify what users should do.

File: sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/OpenTelemetryBuilderExtensions.cs (hunk 1 above)
Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/OpenTelemetryBuilderExtensions.cs:181]: Sorry, i see my mistake. Trying to undo this change: https://github.com/Azure/azure-sdk-for-net/pull/41665/files

Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/OpenTelemetryBuilderExtensions.cs:181]: fixed this. tested and confirmed that Logs are still exported:

image

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md (hunk 2 above)
Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:8]: updated. please rereview

**Review by rajkumar-rangaraj — APPROVED** [review]

**Pull request merged**